.github/workflows/pr-check.yaml

```diff
@@ -275,6 +275,8 @@ jobs:
             sudo apt-get update && \
             sudo apt-get -y install podman; }
           podman version
+          wget https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_22.04/amd64/conmon_2.1.2~0_amd64.deb -O /tmp/conmon_2.1.2.deb
```

Comment:
Would be good to have a comment for this because the rationale is not that easy to guess

Reply:
Done.